library/Zend/Session/SessionManager.php

```diff
@@ -58,9 +58,15 @@ public function __construct(Config\ConfigInterface $config = null, Storage\Stora
      */
     public function sessionExists()
     {
-        $sid = defined('SID') ? constant('SID') : false;
-        if ($sid !== false && $this->getId()) {
-            return true;
+        if (function_exists('session_status')) {
```

Comment:
We could potentially define the constant during the test, and lazy load a file that defines the function, when under PHP 5.3... but ouch. That said, the path is covered when in 5.4+ so, we're good there. The bigger issue is the new test failure. I'll comment on that separately.